src/components/Checkout/LoanPrice.vue

```diff
@@ -75,7 +78,9 @@ export default {
 		};
 	},
 	computed: {
+		/* eslint-disable */
 		prices() {
+			// TODO: convert to switch statement with default value returned
```

Comment:
We could also change that else if to an if since the first if returns something

Reply:
resolved